docs/UserGuide.md

```diff
@@ -213,11 +224,12 @@ tag(s) specified in the input.
 - Supports searching with **multiple tags**.
   - `filter t/Buyer t/Seller` only returns persons with both "Buyer" and "Seller" tags.
 - **Comprehensive searching**, returning all persons' with the specified tag(s).
-    - `filter t/Seller` returns all persons tagged as "Seller".
```

Comment:
why delete uh

Reply:
The scrshot description has exactly the same message